src/components/DefinitionSnippet.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { SearchTerm } from "./SearchResult";
+import { useTerm } from "../api/TermAPI";
+import { Typography } from "@material-ui/core";
+
+const MAX_CHARACTER = 177;
+
+const DefinitionSnippet: React.FC<SearchTerm> = (props) => {
+  const { data = [], isLoading, isSuccess } = useTerm(props);
+  const definition = data.definition?.cs;
+
+  if (isLoading) return <Typography variant="h5">Načítání definice</Typography>;
+  if (isSuccess && definition) {
+    return (
+      <Typography variant="h5">
+        {definition?.length > MAX_CHARACTER
+          ? definition.substr(0, MAX_CHARACTER).concat(" ...")
```

Comment:
Three points here:
1) This could be a utility function outside of the component
2) Instead of the three dots, you probably want to use the ellipsis character: `&hellip;`
3) BUT most importantly, this should be done with CSS :-) The char count is inconsistent because it obviously depends on widths of particular chars. https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/